assets/admin/students/student-bulk-action-button/index.test.js

```diff
@@ -59,4 +49,64 @@ describe( '<StudentBulkActionButton />', () => {
 			)
 		).toBeTruthy();
 	} );
+
+	it( 'Student modal is rendered with action to remove progress on button click when remove progress is selected', () => {
+		setupSelector( [
+			{ value: 'enrol_restore_enrolment' },
+			{ value: 'remove_progress', selected: true },
+			{ value: 'remove_progress' },
```

Comment:
I'm guessing this one should be `remove_enrolment`?

Reply:
[Here](https://github.com/Automattic/sensei/pull/5004/commits/29dfcebc708bef741cd08f5352c45ad49c3fa0f1)